src/display/font_loader.js

```diff
@@ -97,9 +97,17 @@ class FontLoader {
         await fontFace.load();
         this.#systemFonts.add(loadedName);
       } catch {
-        warn(
-          `Cannot load system font: ${loadedName} for style ${style.style} and weight ${style.weight}.`
-        );
+        if (info.mustGuessFallback) {
+          // We're trying to load only one system font.
+          const match = src.match(/^local\((.*)\)$/);
+          warn(
+            `Cannot load system font: ${match[1]}, installing it could help to have a better PDF rendering.`
```

Comment:
Let's ensure that the line below cannot possibly throw, by guarding the `match`-access.
Also, I've suggested a slightly wording tweak:
```suggestion
            `Cannot load system font: ${match?.[1]}, installing it could help improve PDF rendering.`
```